src/ConfigurationsProvider.tsx

```diff
@@ -49,13 +65,23 @@ export const ConfigurationsProvider = ({ children }: { children?: ReactNode }) =
     skip: !Object.keys(sue).length
   });
 
+  const { data: filterData } = useQueryWithApollo(getQuery(QUERY_TYPES.RESOURCE_FILTERS), {
```

Comment:
why i apollo necessary here? i would love to see only react-query in use, as we plan to refactor the whole code base to react-query and get rid of apollo.

Reply:
Yes, that's true, but so far we have only used react-query with endpoints. react-query requires a client to be created first.